CybercentreCanada/superset
# feature/flow-63

### SUMMARY

This pull request introduces a new feature to the Harmonized Email Dashboard (HED) which allows users to manually submit EML files to ASSEMBLYLINE. This feature targets scenarios where email submissions are not automatically handled or when previous submissions have aged out of ASSEMBLYLINE's system. The implementation includes a "Submit to ASSEMBLYLINE" option in the context menu for rows linked to Harmonized Email IDs, enabling users to manually trigger submissions of EML files directly from the dashboard.

### TESTING INSTRUCTIONS

1. Navigate to the Harmonized Email Dashboard.
2. Right-click on a row that represents an email with a Harmonized Email ID.
3. Select "Submit to ASSEMBLYLINE" from the context menu.
4. Verify that the EML file paths are retrieved and submitted correctly to ASSEMBLYLINE via the displayed UI.
5. Check ASSEMBLYLINE to ensure the file appears as expected.
6. Attempt submissions for emails that have aged off, have no attachments, or whose attachments are not present in ASSEMBLYLINE to verify all scenarios.

### ADDITIONAL INFORMATION
